tests/apollo/test_skvbc_preexecution.py

```diff
@@ -369,7 +369,7 @@ async def test_alternate_f_isolated(self, bft_network, tracker):
 
     async def issue_tracked_ops_to_the_system(self, tracker):
         try:
-            with trio.move_on_after(seconds=30):
+            with trio.move_on_after(seconds=40):
```

Comment:
So, your fix is to effectively send more requests? What's the reasoning behind this?

Reply:
From what I've seen, the scenario is as following:
Once we release the first f replicas from the isolation and isolate a different set of f replicas, a view change may be initiated.
If the view change takes too much time the tracker fails from sending any request (because we catch the tooSlowError and pass) and then we end up without creating any new block.

Increasing this timeout gives the tracker more opportunities to issue a successful command.
Although, now that I'm thinking of it, we should try to issue more than 10 requests...

Reply:
I'm wondering though... Don't we rather want to avoid such unexpected view changes? You could increase the view change timeout for tests where we don't want view changes, like here:
https://github.com/vmware/concord-bft/blob/master/tests/apollo/test_skvbc_chaotic_startup.py#L307

Reply:
mmm, a view change is a reasonable scenario in this case, so why preventing it artificially?

Reply:
I'd argue that if the _f_ crashed replicas do not include the primary, we should not have view change. The system should be able to proceed with the same primary, even with _f_ crashed nodes...

Reply:
They did not crash, in this test we isolate a group A of f replicas, and check the system is able to make progress.
Then we release group A and isolate a distinct group B of f replicas and check that the system is able to make progress.

So it makes sense that sometimes group A will try to perform a view change

And when I'm thinking of it now, if only f replicas are trying to make a view change they should not succeed, but the remaining f+1 replicas will also get stack and they will initiate a view change.
So we will have view change in this test in anyway, we just want to send enough messages such that we will see progress eventually (we can even wait for sure for a view change)

Reply:
Well, it depends how you isolate group A :) If they are also isolated from client requests, there won't be view change.

Reply:
Yeah, you write...
I will increase the viewchange timeout